monitoring/collector/sysctl.go

```diff
@@ -0,0 +1,86 @@
+/*
+Copyright 2017 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package collector
+
+import (
+	"strconv"
+	"sync"
+
+	"github.com/gravitational/satellite/monitoring"
+	"github.com/gravitational/trace"
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+// SysctlCollector collects metrics from kernel system params
+type SysctlCollector struct {
+	ipv4Forwarding typedDesc
+	brNetfilter    typedDesc
+	mutex          sync.RWMutex
+}
+
+// NewSysctlCollector returns an initialized SysctlCollector.
+func NewSysctlCollector() Collector {
```

Comment:
same here - strive to return a concrete type - this has no effect on external callers